resources/artifact-ignores.properties

```
@@ -715,3 +715,7 @@ starteam = https://github.com/jenkins-infra/update-center2/pull/571
 
 # https://www.jenkins.io/security/advisory/2022-03-29/
 phoenix-autotest = https://www.jenkins.io/security/plugins/#suspensions
+
+# Obsolete and no longer used by Jenkins X, see https://github.com/jenkins-x/jx/issues/7020
+jx-pipelines
+jx-resources
```

```suggestion
jx-pipelines = https://github.com/jenkins-infra/update-center2/pull/590
jx-resources = https://github.com/jenkins-infra/update-center2/pull/590
```
to provide a warning for existing users